Modules/Net/settings.swift

```diff
@@ -123,7 +123,7 @@ internal class Settings: NSView, Settings_v {
     @objc func handleSelection(_ sender: NSPopUpButton) {
         guard let item = sender.selectedItem else { return }
         
-        if item.title == "Autodetection" {
+        if item.title == LocalizedString("Autodetection") {
```

Comment:
The same problem as in the Battery module.

Reply:
https://github.com/exelban/stats/blob/e33082f0cab3b2ea171c4cb06848c459843f6897/Modules/Net/settings.swift#L126-L127
When `item.title == "Autodetection"`, the term `Network_interface` would just be removed, so I think saving state would not be changed.